core/object/script_language.cpp

```diff
@@ -480,6 +484,16 @@ StringName ScriptServer::get_global_class_native_base(const String &p_class) {
 	return base;
 }
 
+bool ScriptServer::is_global_class_abstract(const String &p_class) {
+	ERR_FAIL_COND_V(!global_classes.has(p_class), false);
+	return global_classes[p_class].is_abstract;
```

Comment:
```suggestion
	GlobalScriptClass *global_class = global_classes.getptr(p_class);
	ERR_FAIL_COND_V(!global_class, false);
	return global_class->is_abstract;
```
Same below.

Reply:
Unresolved.

Reply:
Oh, I forgot.

However, on this new inspection, I realize why I did it like that: for consistency, to match how all the other `ScriptServer::get_global_class_*()` getters are made. A change like the one you're suggesting would be ideal for a subsequent PR that addresses them all at once.